openid-connect-server/src/main/java/org/mitre/openid/connect/service/impl/MITREidDataService_1_1.java

```diff
@@ -1218,7 +1218,7 @@ private void fixObjectReferences() {
             }
             Long newGrantId = grantOldToNewIdMap.get(oldGrantId);
             ApprovedSite site = approvedSiteRepository.getById(newGrantId);
-            site.setApprovedAccessTokens(tokens);
+//            site.setApprovedAccessTokens(tokens);
```

Comment:
Same as the 1.0 note: you need to link up the approved sites and tokens appropriately.